### ISSUE TITLE
Meeting record for the meeting creator and to download it

### ISSUE THREAD

**Issue by pgwaruna** [fmeringdal/nettu-meet#33]
IT would be great to have a feature to record the meeting and to download the video in compressed format to the meeting scheduling person

**Comment by fmeringdal** [comment]
Yep, that would be a great improvement. It will be take some effort to implement that, but it certainly is in the backlog.
